css/_menus.scss

```diff
@@ -169,11 +167,14 @@
         top: 100%;
         right: 100%;
         margin: -40px $full-padding*2 0 0;
-		text-transform: lowercase;
 
         &:before {
             left: 100%;
             margin-top: $full-padding*2;
         }
     }
 }
+
+.context-btn > span > ul > li {
```

Comment:
Following [Writing efficient CSS](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Writing_efficient_CSS#Avoid_the_descendant_selector.21), could you avoid using the descendant selector?